cmd/pulumi-test-language/test_host.go

```diff
@@ -229,6 +229,12 @@ func (h *testHost) SignalCancellation() error {
 }
 
 func (h *testHost) Close() error {
+	for _, closer := range h.connections {
+		if err := closer.Close(); err != nil {
+			return fmt.Errorf("close provider connection: %w", err)
```

Comment:
Should we try to finish closing the other connections before returning the error?

Reply:
yeh error join is probably a better thing here.